examples/minimal.rs

```diff
@@ -98,8 +101,10 @@ async fn main() {
     let opts = Options::from_args();
     init_log(opts.verbosity);
 
-    if opts.count <= 1 {
-        start_single_node(opts.first, opts.bootstrap_contacts, opts.ip, opts.port).await
+    let handlers: Vec<JoinHandle<()>> = if opts.count <= 1 {
```

Comment:
Also, it's `JoinHandle`, not `JoinHandler`, so the variable should probably be called `handles`.